apps/automated-tests/tests/e2e/checkout/logIn.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test } from "fixtures/fixtures";
+import { URLS, userEmail, userPassword } from "utils/constants";
+
+test.describe(
+  "LogIn",
+  {
+    tag: "@CI",
+  },
+  () => {
+    const testEnvUrl = process.env.TEST_ENV_URL ?? ""; //get testEnvUrl from .env
```

Comment:
No need for this comment in my opinion, the code is pretty straightforward

```suggestion
    const testEnvUrl = process.env.TEST_ENV_URL ?? "";
```